src/github.com/matrix-org/dendron/login/database.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 type database interface {
-	passwordHash(userID string) (string, error)
+	passwordHash(userID string) (string, string, error)
```

Comment:
Document what this returns, given it returns two things of the same type and its name doesn't at all indicate what they are